src/main/kotlin/de/itemis/mps/gradle/runmigrations/Plugin.kt

```diff
@@ -0,0 +1,83 @@
+package de.itemis.mps.gradle.runmigrations
+
+import de.itemis.mps.gradle.*
+import net.swiftzer.semver.SemVer
+import org.gradle.api.GradleException
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Copy
+import org.gradle.kotlin.dsl.withGroovyBuilder
+import java.io.File
+import javax.inject.Inject
+
+open class MigrationExecutorPluginExtensions @Inject constructor(of: ObjectFactory, project: Project) : BasePluginExtensions(of, project) {
+    val force: Property<Boolean> = of.property(Boolean::class.java).convention(false)
+}
+@Suppress("unused")
+open class RunMigrationsMpsProjectPlugin : Plugin<Project> {
+    override fun apply(project: Project) {
+        project.run {
+            val extension = extensions.create("runMigrations", MigrationExecutorPluginExtensions::class.java)
+            afterEvaluate {
+                val mpsLocation = extension.mpsLocation.convention{ File(project.buildDir, "mps") }.map { it.asFile }.get()
+                val projectLocation = extension.projectLocation.getOrElse { throw GradleException("No project path set") }
```

Comment:
The docs in README.md say projectLocation is optional yet here an error is thrown if it's not set?

Reply:
I've fixed the readme.